explorer/impact.go

```diff
@@ -11,6 +11,15 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// Impact represents severity rating scale when impact is provided as human-readable string value
+var impactMapping = map[string]int32{
+	"none":     0,
+	"low":      10,
+	"medium":   40,
+	"high":     70,
+	"critical": 100,
```

Comment:
I'd recommend to place the numbers in the middle of each of the ranges. Currently critical is on the top end of the range, while the others are on the low end. I think it would help better balance things. (That said, I could see the argument to leave critical the way you have it)

Reply:
Right now, I used always the lowest number except for critical because it felt strange to have 90 for critical. I see both options: lowest or middle values working. I would appreciate if you could propose the middle values so I can just adjust to those.

Reply:
The middle would be: 95 critical, 80 high, 55 medium, 20 low, 0 none

Reply:
The problem with having the lower values on everything is that it skews the calculation on all evaluations this way. I.e. it's very easy to have something that is high (low end 70) have a much too lower influence on the overall calculation (of an asset or a policy) because it is at the very bottom end of the scale. That's why I'd recommend against the lower end of the scale. Btw same reason applies to the top end of the scale. Third reason to pick the middle is to give us some wiggle room to adjust severities in policies within each band to prioritize things higher/lower